src/ir.cpp

```diff
@@ -21049,6 +21049,12 @@ static IrInstGen *ir_analyze_instruction_elem_ptr(IrAnalyze *ira, IrInstSrcElemP
 
     bool safety_check_on = elem_ptr_instruction->safety_check_on;
     if (instr_is_comptime(casted_elem_index)) {
+        if ((err = ir_resolve_const_val(ira->codegen, ira->new_irb.exec,
```

Comment:
Small nit, the pattern we usually use is:
```zig
        ZigValue *val = ir_resolve_const(ira, casted_elem_index, UndefBad);
        if (val == nullptr)
            return ira->codegen->invalid_inst_gen;
```

Reply:
thanks